src/mars-photos.js

```diff
@@ -19,6 +19,8 @@ export default function marsPhotos(): Object {
       curiosity: ['fhaz', 'rhaz', 'mast', 'chemcam', 'mahli', 'mardi', 'navcam'],
       opportunity: ['fhaz', 'rhaz', 'navcam', 'pancam', 'minites'],
       spirit: ['fhaz', 'rhaz', 'navcam', 'pancam', 'minites'],
+      // eslint-disable-next-line max-len
+      perseverance: ['edl_rucam', 'edl_rdcam', 'edl_ddcam', 'edl_pucam1', 'edl_pucam2', 'navcam_left', 'navcam_right', 'mcz_right', 'mcz_left', 'front_hazcam_left_a', 'front_hazcam_right_a', 'rear_hazcam_left', 'rear_hazcam_right'],
```

Comment:
we should update https://github.com/AJFunk/nasa-sdk/blob/master/tests/mars-photos.spec.js with quick specs for the new `perserverance` option